### DISCUSSION TITLE
Two questions about firefly iii

### DISCUSSION THREAD

**Discussion by pedrom34** [firefly-iii/firefly-iii#4531] [General]
I really like the new favicon 👌 
Is it possible to have a high res version of it? I want to create an icon for my 2FA app (aegis).

My other question is: why "firefly-III"? I mean, why this name @JC5 ?

Thanks 😄

**Comment by JC5** [comment]
Glad to hear you like it! I haven't got the high-res version available for you unfortunately.

For some background on the name, here's an old [reddit thread](https://old.reddit.com/r/FireflyIII/comments/er5443/backstory/) with some info!

**Reply by pedrom34** [reply]
Thanks for the backstory!
For the image, is it because you want to keep it to yourself? If that's the case, I understand. Or is it because you only have it in 32x32?

**Reply by JC5** [reply]
I want to keep the high-res for myself. It was designed specifically for Firefly III and I'd like to to gain some traction before I start dropping high-res versions.

**Reply by pedrom34** [reply]
I totally understand 👍 
Anyway, I'm sure all the users will love this new identity! Keep up the good work!
